### PR TITLE
feat(web): Add Mideind contentful translation app to codebase

### PR DESCRIPTION
**Pull request by RunarVestmann**
# Add Mideind contentful translation app to codebase

## What

* The Mideind contentful extension has now been moved into an app format (which is the new way to extend functionality within Contentful)

## Why

* We can't use newer features (like Compose) unless we move older Contentful extensions into the app format

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by busla — APPROVED** [review]
LGTM

**Review by kksteini — APPROVED** [review]

File: apps/contentful-apps/components/MideindTranslationSidebar/api/index.tsx [hunk 1]
```diff
@@ -1,22 +1,26 @@
-const base = 'https://stafraentisland.greynir.is/translate'
+import getConfig from 'next/config'
+
+const baseUrl = 'https://stafraentisland.greynir.is/translate'
```
Comment (siggifv) [inline apps/contentful-apps/components/MideindTranslationSidebar/api/index.tsx:3]: I'd like to see this stored in the config.

**Review by siggifv — APPROVED** [review]
Approved, with on comment regarding storing the base URL in config.

**Review by eirikurn — APPROVED** [review]
Nice!

**Pull request merged**